app/code/Mediotype/Employee/composer.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "mediotype/employee",
```

```suggestion
    "name": "mediotype/module-employee",
```

More in line with how Magento does stuff.  I think they also have some code that parses for "module-*"